diff -r 9a486764cba7 RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/IShoppingCart.java
--- a/RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/IShoppingCart.java	Thu Aug 01 16:02:18 2013 +0300
+++ b/RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/IShoppingCart.java	Fri Aug 02 12:13:09 2013 +0300
@@ -30,7 +30,7 @@
     @PUT
     @Path("/update")
     @Consumes(MediaType.APPLICATION_JSON)
-    Response updateTicket(@Context HttpServletRequest request, Item item, int quantity);
+    Response updateItem(@Context HttpServletRequest request, Item item, int quantity);
 
     @DELETE
     @Path("/delete")
diff -r 9a486764cba7 RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/Item.java
--- a/RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/Item.java	Thu Aug 01 16:02:18 2013 +0300
+++ b/RestShoppingCartTask/ShoppingCartApi/src/main/java/com/skrill/interns/shoppingCartApi/Item.java	Fri Aug 02 12:13:09 2013 +0300
@@ -10,19 +10,16 @@
 public class Item {
 	private String name;
 	private Integer quantity;
-	private static Integer id;
 
 	public Item() {
-		id++;
 	}
 
 	public Item(String name, Integer quantity) {
 		this.name = name;
 		this.quantity = quantity;
-		id++;
 	}
 
-	@XmlElement(nillable = true, required = false)
+	@XmlElement(nillable = false, required = true)
 	public String getName() {
 		return name;
 	}
@@ -41,9 +38,4 @@
 		this.quantity = quantity;
 	}
 
-	@XmlElement(nillable = false, required = false)
-	public Integer getID() {
-		return id;
-	}
-
 }
